go.mod

```diff
@@ -0,0 +1,7 @@
+module github.com/okteto/deploy-preview
+
+go 1.22
+
+require golang.org/x/text v0.16.0
+
+require github.com/hoshsadiq/godotenv v1.0.0 // indirect
```

Comment:
I used my custom fork of https://github.com/joho/godotenv/ which parses things a lot better. But happy to switch it back to https://github.com/joho/godotenv/, or another one if you have a preference.